types/base.d.ts

```diff
@@ -1,6 +1,9 @@
+import { Message } from "./message";
+
 export declare class API {
-    constructor();
-    load(): Promise<void>;
+    constructor(options?: Record<string, unknown>);
+    static load(): Promise<void>;
+    trigger(topic: string, message: Message, options?: Record<string, unknown>): Promise<void>;

```

Comment:
In javascript classes we always follow the same order for this stuff: https://github.com/hivesolutions/yonius/blob/727e8a5ab3268aa4d5c3625bf345c1732906551a/types/data/model.d.ts

Reply:
Ok, I was following the same order as they appeared defined in the base API class. Changing that.

Reply:
Well you got me thinking I must admit.